src/plugins/intel_gpu/include/intel_gpu/runtime/layout.hpp

```diff
@@ -371,6 +371,12 @@ struct layout {
 
     layout(const layout& other) = default;
 
+    layout()
+        : data_type(cldnn::data_types::bin)
+        , format(cldnn::format::any)
+        , data_padding(padding())
+        , size(ov::PartialShape()) { }
```

Comment:
Why do we need this constructor ?

Reply:
This is for deserialization. But I removed it from this PR. Thank you!